app.rb

```diff
@@ -1,7 +1,33 @@
 require 'sinatra/base'
+require './lib/player'
+require './lib/computer'
+require './lib/game'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+	enable :sessions
+
+	 get '/' do
+    erb(:index)
   end
+
+   post '/name' do 
+  	session[:player] = params[:player]
+    redirect '/play'
+  end
+
+  get '/play' do 
+  	@player = session[:player]
+  	@weapon = session[:weapon]
+  	@computer_weapon = session[:computer_weapon]
+  	erb(:play)
+  end
+
+  post '/play' do 
+  	session[:weapon] = params[:weapon]
+  	session[:computer_weapon] = 'scissors'
```

Comment:
I think it makes a lot of sense to hard code a response for computer whilst you are working on the game logic - nice one! ⭐️ 

What would you do to randomise this?